surprise/prediction_algorithms/knns.py

```diff
@@ -254,10 +254,11 @@ class KNNBaseline(SymmetricAlgo):
 
     """
 
-    def __init__(self, k=40, min_k=1, sim_options={}, bsl_options={}):
+    def __init__(self, k=40, min_k=1, sim_options={}, bsl_options={},
+                    **kwargs):
```

Comment:
`**kwargs` needs to be aligned just after the first `(`.

(i.e. with the `s` char)